notification2-examples/hello-world-microservice/pom.xml

```diff
@@ -3,21 +3,20 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>c8y.example.notification</groupId>
```

Comment:
thought: (maybe suggestion) should the groupId be c8y.example.notification2 (with a 2 on the end, to avoid any potential for association with the standard notifications)

Reply:
It is clear from the artifact id and besides notification2 is only game in town 😉

Reply:
Maybe it's just my preference to say it again. I won't argue for it though.